Samples/Samples.Android/Properties/AndroidManifest.xml

```diff
@@ -12,5 +12,11 @@
   <uses-feature android:name="android.hardware.location" android:required="false" />
   <uses-feature android:name="android.hardware.location.gps" android:required="false" />
   <uses-feature android:name="android.hardware.location.network" android:required="false" />
-  <application android:label="@string/app_name" android:icon="@mipmap/ic_launcher" android:roundIcon="@mipmap/ic_launcher_round" android:theme="@style/MainTheme"></application>
+  <application android:label="@string/app_name" 
+               android:icon="@mipmap/ic_launcher" 
+               android:roundIcon="@mipmap/ic_launcher_round" 
+               android:theme="@style/MainTheme"
+               android:fullBackupContent="@xml/my_backup_rules">
```

Comment:
Can we have our rules and the user rules together? This appears to be an exclusive resource, so we may have to just let the user know - see my other comment.